Index: new/config/tools/xml/cluster.rng.in.head
===================================================================
--- new.orig/config/tools/xml/cluster.rng.in.head	2012-10-02 10:58:00.000000000 +0200
+++ new/config/tools/xml/cluster.rng.in.head	2012-10-02 11:07:54.000000000 +0200
@@ -998,6 +998,9 @@
    <zeroOrMore>
     <ref name="VM"/>
    </zeroOrMore>
+   <zeroOrMore>
+    <ref name="PVEVM"/>
+   </zeroOrMore>
   </interleave>
   </element>
  </optional>
