
 * check if cman_tool is installed

Index: new/cman/init.d/cman.in
===================================================================
--- new.orig/cman/init.d/cman.in	2012-06-13 15:50:21.000000000 +0200
+++ new/cman/init.d/cman.in	2012-06-13 15:55:37.000000000 +0200
@@ -22,6 +22,8 @@
 # save invokation for rollback ops
 thisinvokation="$0"
 
+test -x @SBINDIR@/cman_tool || exit 0
+
 chkconfig2()
 {
 	case "$1" in
@@ -138,7 +140,7 @@
 #     When setting FENCE_JOIN to "no", it is important to check 
 #     DLM_CONTROLD_OPTS to reflect expected behavior regarding fencing
 #     and quorum.
-[ -z "$FENCE_JOIN" ] && FENCE_JOIN="yes"
+[ -z "$FENCE_JOIN" ] && FENCE_JOIN="no"
 
 # FENCED_OPTS -- allow extra options to be passed to fence daemon.
 [ -z "$FENCED_OPTS" ] && FENCED_OPTS=""
