
 * check if cman_tool is installed

Index: new/cman/init.d/cman.in
===================================================================
--- new.orig/cman/init.d/cman.in	2013-03-15 07:21:53.000000000 +0100
+++ new/cman/init.d/cman.in	2013-03-15 07:27:17.000000000 +0100
@@ -19,6 +19,8 @@
 # set secure PATH
 PATH="/bin:/usr/bin:/sbin:/usr/sbin:@SBINDIR@"
 
+test -x @SBINDIR@/cman_tool || exit 0
+
 chkconfig2()
 {
 	case "$1" in
@@ -143,7 +145,7 @@
 #     DLM_CONTROLD_OPTS="-f0" (at least) for correct operation.
 #     Please note that clusters without fencing are not 
 #     supported by Red Hat except for MRG installations.
-[ -z "$FENCE_JOIN" ] && FENCE_JOIN="yes"
+[ -z "$FENCE_JOIN" ] && FENCE_JOIN="no"
 
 # FENCED_OPTS -- allow extra options to be passed to fence daemon.
 [ -z "$FENCED_OPTS" ] && FENCED_OPTS=""
