Index: new/rgmanager/init.d/rgmanager.in
===================================================================
--- new.orig/rgmanager/init.d/rgmanager.in	2012-06-14 17:38:33.000000000 +0200
+++ new/rgmanager/init.d/rgmanager.in	2012-06-15 11:41:09.000000000 +0200
@@ -6,10 +6,10 @@
 #
 ### BEGIN INIT INFO
 # Provides:		rgmanager
-# Required-Start:	cman cpglockd
-# Required-Stop:	cman cpglockd
-# Default-Start:
-# Default-Stop:
+# Required-Start:	$remote_fs cman
+# Required-Stop:	$remote_fs cman
+# Default-Start:        2 3 4 5
+# Default-Stop:         0 1 6
 # Short-Description:	Starts and stops Red Hat Service (resource group) Manager
 # Description:		Starts and stops Red Hat Service (resource group) Manager
 ### END INIT INFO
@@ -20,6 +20,10 @@
 # set secure PATH
 PATH="/sbin:/bin:/usr/sbin:/usr/bin:@SBINDIR@"
 
+CCFG="/etc/cluster/cluster.conf"
+
+test -f "$CCFG" || exit 0
+
 success()
 {
 	echo -ne "[  OK  ]\r"
@@ -52,8 +56,7 @@
 
 # deb based distros
 if [ -d /etc/default ]; then
-	[ -f /etc/default/cluster ] && . /etc/default/cluster
-	[ -f /etc/default/rgmanager ] && . /etc/default/rgmanager
+	[ -f /etc/default/redhat-cluster-pve ] && . /etc/default/redhat-cluster-pve
 	[ -z "$LOCK_FILE" ] && LOCK_FILE="/var/lock/rgmanager"
 fi
 
@@ -82,6 +85,11 @@
 		while ! cpglockdump > /dev/null 2>&1; do sleep 1; done
 }
 
+rgmanager_enabled()
+{
+	ccs_tool query /cluster/rm >/dev/null 2>&1
+}
+
 rtrn=0
 
 if [ "$EUID" != "0" ]; then
@@ -91,6 +99,11 @@
 
 case "$1" in
 start)
+
+	if ! rgmanager_enabled; then
+	    exit 0;
+	fi
+
 	echo -n "Starting $ID: "
 
 	# most recent distributions use tmpfs for /var/run
@@ -107,7 +120,9 @@
 	if status $RGMGRD > /dev/null 2>&1; then
 		success
 	else
-		if $RGMGRD $RGMGR_OPTS; then
+	        # do not trust return code
+		$RGMGRD $RGMGR_OPTS;
+		if status $RGMGRD > /dev/null 2>&1; then
 			touch $LOCK_FILE
 			success
 		else
@@ -141,11 +156,12 @@
 	echo -n "Stopping $ID: "
 
 	if status $RGMGRD > /dev/null 2>&1; then
-		if stop_cluster; then
-			success
-		else
-			failure
+	        stop_cluster
+		if status $RGMGRD > /dev/null 2>&1; then
+		        failure
 			rtrn=1
+	        else
+		        success
 		fi
 	else
 		success
