Index: new/cman/init.d/cman.in
===================================================================
--- new.orig/cman/init.d/cman.in	2012-06-13 15:55:37.000000000 +0200
+++ new/cman/init.d/cman.in	2012-06-13 15:56:41.000000000 +0200
@@ -8,10 +8,10 @@
 #
 ### BEGIN INIT INFO
 # Provides:		cman
-# Required-Start:	$network $time
-# Required-Stop:	$network $time
-# Default-Start:
-# Default-Stop:
+# Required-Start:	$remote_fs pve-cluster
+# Required-Stop:	$remote_fs pve-cluster
+# Default-Start:        2 3 4 5
+# Default-Stop:         0 1 6
 # Short-Description:	Starts and stops cman
 # Description:		Starts and stops the Cluster Manager set of daemons
 ### END INIT INFO
@@ -22,7 +22,10 @@
 # save invokation for rollback ops
 thisinvokation="$0"
 
+CCFG="/etc/cluster/cluster.conf"
+
 test -x @SBINDIR@/cman_tool || exit 0
+test -f "$CCFG" || exit 0
 
 chkconfig2()
 {
@@ -71,8 +74,7 @@
 
 # deb based distros
 if [ ! -d /etc/sysconfig ]; then
-	[ -f /etc/default/cluster ] && . /etc/default/cluster
-	[ -f /etc/default/cman ] && . /etc/default/cman
+	[ -f /etc/default/redhat-cluster-pve ] && . /etc/default/redhat-cluster-pve
 	[ -z "$LOCK_FILE" ] && LOCK_FILE="/var/lock/cman"
 	netmanager=network-manager
 fi
