
 * check if cman_tool is installed

Index: new/cman/init.d/cman.in
===================================================================
--- new.orig/cman/init.d/cman.in	2010-12-02 07:19:35.000000000 +0100
+++ new/cman/init.d/cman.in	2010-12-23 11:32:12.000000000 +0100
@@ -19,6 +19,8 @@
 # set secure PATH
 PATH="/bin:/usr/bin:/sbin:/usr/sbin:@SBINDIR@"
 
+test -x @SBINDIR@/cman_tool || exit 0
+
 chkconfig2()
 {
 	case "$1" in
